{{ cookiecutter.target }}/etc/zope.conf

```diff
@@ -5,8 +5,8 @@ instancehome $INSTANCEHOME
 %define CLIENTHOME ABSPATH({{ cookiecutter.location_clienthome }})
 clienthome $CLIENTHOME
 
-debug-mode {{ "on" if cookiecutter.debug_mode == "True" else "off" }}
-security-policy-implementation {{ "PYTHON" if cookiecutter.verbose_security == "True" else "C"}}
+debug-mode {{ "on" if cookiecutter.debug_mode else "off" }}
```

Comment:
This will break existing installations and would set these options to `on` where people used `debug_mode: off` (was already broken) or `debug_mode: 'False'`, right?

Since this is a breaking change and people are using it already to develop and deploy Plone we should be more cautious.

What do you think about a helper method like:

```
def cast_boolean(value) -> Boolean:
    return value in ("on", "True", True)
```

Reply:
yes, thats the intention of a major version cleanup

Reply:
If we cast/fallback all of them in the templates, a Jinja2  filter would be the way.

Reply:
better idea: fail in pre-hook.

Reply:
it checks now if debug_mode or verbose_security is boolean, if not it fails with explaining message.